From 24c4ca96201806d81f9dbda409a5c9e892f1b300 Mon Sep 17 00:00:00 2001
From: Keith Gable <gablk@amazon.com>
Date: Fri, 26 May 2023 18:14:39 -0700
Subject: [PATCH] RHEL 8.8 backported the same change as RHEL 9.2, so the check
 that is needed for RHEL 9.2 is also needed for RHEL 8.8.

See: https://github.com/amzn/amzn-drivers/issues/271
---
 ena/kcompat.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ena/kcompat.h b/ena/kcompat.h
index 1b3e7ed..f6ea16b 100644
--- a/ena/kcompat.h
+++ b/ena/kcompat.h
@@ -1093,7 +1093,7 @@ static inline void ena_netif_napi_add(struct net_device *dev,
 				      int (*poll)(struct napi_struct *, int))
 {
 #if (LINUX_VERSION_CODE < KERNEL_VERSION(6, 1, 0)) && \
-	!(RHEL_RELEASE_CODE && (RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(9, 2)))
+	!(RHEL_RELEASE_CODE && (RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(9, 2) || RHEL_RELEASE_CODE == RHEL_RELEASE_VERSION(8, 8)))
 #ifndef NAPI_POLL_WEIGHT
 #define NAPI_POLL_WEIGHT 64
 #endif
-- 
2.32.1 (Apple Git-133)

